### ISSUE TITLE
When installed inside Tox: UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 388: ordinal not in range(128)

### ISSUE THREAD

**Issue by balloob** [zopefoundation/RestrictedPython#71]
Some Home Assistant developers have reported an issue installing RestrictedPython 4.0a2 inside tox because of a UnicodeDecode error when `setup.py` reads the README file. Our issue is here: https://github.com/home-assistant/home-assistant/issues/8026

Installing outside Tox does not give us any problems.

```
Collecting restrictedpython==4.0a2 (from -r /home/alex/home-assistant/requirements_test_all.txt (line 107))
  Using cached RestrictedPython-4.0a2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-faptnbow/restrictedpython/setup.py", line 33, in <module>
        long_description=(read('README.rst') + '\n' +
      File "/tmp/pip-build-faptnbow/restrictedpython/setup.py", line 23, in read
        return open(os.path.join(os.path.dirname(__file__), *rnames)).read()
      File "/home/alex/home-assistant/.tox/py36/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 388: ordinal not in range(128)
 ```

We set our language inside tox to UTF-8 ([source](https://github.com/home-assistant/home-assistant/blob/dev/tox.ini#L8-L13)):

```
install_command = /usr/bin/env LANG=C.UTF-8 pip install {opts} {packages}
```

I guess a potential solution could be to pass in `encoding='utf8'` to the open call.

**Comment by icemac** [comment]
@balloob I removed the non ASCII character from README.rst and released a version 4.0a3 to PyPI. Please check if this fixes the error you encountered.

**Comment by balloob** [comment]
Thanks! 🎉  I have added the updated dependency to Home Assistant and have asked the person experiencing the issue to report here if it has been fixed.
